src/plugin/manager/email/category_manager.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from spaceone.inventory.plugin.collector.lib import *
+
+from plugin.conf.cloud_service_conf import AUTH_TYPE
+from plugin.connector.email.category_connector import CategoryConnector
+from plugin.manager.base import NHNCloudBaseManager
+
+_LOGGER = logging.getLogger("cloudforet")
+
+
+class CategoryManager(NHNCloudBaseManager):
+    auth_type = AUTH_TYPE.APP_KEY
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.cloud_service_group = "Email"
+        self.cloud_service_type = "Category"
+        self.provider = "NHNCloud"
+        self.metadata_path = "metadata/email/category.yaml"
+
+    def create_cloud_service_type(self):
+        cloud_service_type = make_cloud_service_type(
+            name=self.cloud_service_type,
+            group=self.cloud_service_group,
+            provider=self.provider,
+            metadata_path=self.metadata_path,
+            is_primary=True,
+            is_major=True,
```

Comment:
Add tags for icon

See reference:
https://github.com/cloudforet-io/plugin-aws-inven-collector/blob/master/src/plugin/manager/ec2/instance_manager.py

There is a static image repository for plugin
See: https://github.com/cloudforet-io/static-assets

Reply:
In email service group, what kinds of cloud service type exists?

In that group, there is one "is_primary=True" which is the most important information in this group.

Reply:
@choonho I agreee that category is not primary resource in Email cloud service group.
I expect that received emails are primary resource in Email cloud service group. But I can't sure that received email should be in cloudforet. NHN Cloud do have an API for listing recieved email.

Reply:
Cloud service types below are in Email cloud service group.
email template, tag, UID, category

Reply:
Is it ok to have no primary resource in cloud service group? @choonho

Reply:
@choonho 
I'll just add the email icon first. We will add the rest of the images later.
https://github.com/cloudforet-io/static-assets/pull/1

Reply:
You have to consider what kinds of email information is needed at Cloudforet. Cloudforet focuses on observabilities of resources, not creating/deleting/updating. So listing the cloud service types in Email, then choose the most important item as primary resource.

Reply:
As a cloud operator of NHN Cloud, do you need to manage Email information? The operator is not email user. If you want to integrate email service, you have to think as email administrator.

Reply:
I think resources like received email is not necessary for cloud admin.
But like SES in aws, related resources like template and email categories could be managed by cloud admin because it is actually managed by nhn cloud console now. @choonho